styles/2-middle.css

```diff
@@ -79,6 +80,67 @@ main {
   line-height: 16px;
 }
 
+/* Work pop-up */
+.work-popup-container {
+  position: fixed;
+  top: 0;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  background-color: rgba(170, 170, 170, 0.86);
+  backdrop-filter: blur(4px);
+  width: 100%;
+  height: 100vh;
+  padding: 20px 16px;
+}
+
+.work-popup-bg {
+  position: relative;
+  background-color: white;
+  max-width: 450px;
+  border-radius: 16px;
+}

```

Comment:
- [ ] [Optional] In the pop-up window, I can only see the image and the close button, I had to zoom out to 50% before I can see the whole pop-up, and it's not scrollable. Kindly fix this. you can make it scrollable or you adjust the image size so all the project details can be seen.
![Capture](https://user-images.githubusercontent.com/85095702/162191467-dd828bc7-abb4-4cff-a568-d9d35330e547.PNG)